elmslie-core/src/main/java/vivid/money/elmslie/core/store/linking/LinkingExt.kt

```diff
@@ -0,0 +1,69 @@
+package vivid.money.elmslie.core.store.linking
+
+import vivid.money.elmslie.core.store.Store
+
+fun <
+    ParentEvent : Any,
+    ParentState : Any,
+    ParentEffect : Any,
+    ChildEvent : Any,
+    ChildState : Any,
+    ChildEffect : Any,
+> Store<ParentEvent, ParentState, ParentEffect>.linkTo(
+    childStore: Store<ChildEvent, ChildEffect, ChildState>,
+    effectMapper: (ChildEffect) -> ParentEvent?,
+    stateMapper: (ChildState) -> ParentEvent?,
+) = apply {
+    launch {
+        childStore.states().collect { stateMapper.invoke(it)?.let(this@linkTo::accept) }
+        childStore.effects().collect { effectMapper.invoke(it)?.let(this@linkTo::accept) }
```

Comment:
effects won't be collected( should launch collecting in separate coroutines

Reply:
got it